e2e/_suites/ingest-manager/features/fleet_mode_agent.feature

```diff
@@ -21,6 +21,17 @@ Scenario: Stopping the agent stops backend processes
   Then the "filebeat" process is in the "stopped" state on the host
     And the "metricbeat" process is in the "stopped" state on the host
 
+@restart-agent
+Scenario Outline: Restarting a <os> agent restarts backend processes
+  Given an agent running on "<os>" is deployed to Fleet
+  When the host is restarted
+  Then the "elastic-agent" process is in the "started" state on the host
+    And the "filebeat" process is in the "started" state on the host
+    And the "metricbeat" process is in the "started" state on the host
+Examples:
+| os             |
+| centos-systemd |
```

Comment:
We need `centos-system` as it's a specific version of Centos supporting systemd as pid 1